libr/util/idpool.c

```diff
@@ -163,6 +162,65 @@ R_API void* r_id_storage_get(RIDStorage* storage, ut32 id) {
 	return storage->data[id];
 }
 
+R_API bool r_id_storage_get_lowest(RIDStorage *storage, ut32 *id) {
+	r_return_val_if_fail (storage, false);
+	int fd = 0;
+	if (!r_id_storage_get_prev (storage, (ut32 *)&fd) && fd == 0) {
+		return false;
+	}
+	if (!r_id_storage_get_next (storage, (ut32 *)&fd) && fd == 0) {
+		return false;
+	}
+	*id = fd;
+	return true;
+}
+
+R_API bool r_id_storage_get_highest(RIDStorage *storage, ut32 *id) {
```

Comment:
btw @condret , this really implies that the "highest" id is the one with a bigger value and that the "lowest" is the one with the smaller value. Really, all this just says that there is an ascending order. And that is also confirmed by `o`, which always shows stuff in ascending order.

Reply:
agree